x-pack/plugins/fleet/public/applications/fleet/sections/agents/agent_list_page/components/view_errors.tsx

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { stringify } from 'querystring';
+
+import styled from 'styled-components';
+import React from 'react';
+import { encode } from '@kbn/rison';
+import type { EuiBasicTableProps } from '@elastic/eui';
+import { EuiButton, EuiAccordion, EuiToolTip, EuiText, EuiBasicTable } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { RedirectAppLinks } from '@kbn/shared-ux-link-redirect-app';
+
+import { i18n } from '@kbn/i18n';
+
+import type { ActionErrorResult } from '../../../../../../../common/types';
+
+import { buildQuery } from '../../agent_details_page/components/agent_logs/build_query';
+
+import type { ActionStatus } from '../../../../types';
+import { useStartServices } from '../../../../hooks';
+
+const TruncatedEuiText = styled(EuiText)`
```

Comment:
nit: could we use the `eui-textTruncate` class here instead of a custom style?

Reply:
tried, though it doesn't work well with the auto sized table, the third column becomes invisible:

<img width="651" alt="image" src="https://user-images.githubusercontent.com/90178898/223444580-9d17fe87-3704-4964-b111-6b072e4b0c92.png">